### PR TITLE
Cleaning history - part I

### PR DESCRIPTION
**Pull request by IhateTrains**

### CODE REVIEW

**Comment by IhateTrains** [comment]
Validator errors now: 8480

File: WTWSMS/localisation/WtWSMS_misc.csv [hunk 1]
```diff
@@ -197,3 +197,6 @@ priscillian_carnivore;Carnivourous Clergy;;;;;;;;;;;;;x
 priscillian_carnivore_desc;This Priscillianist priest, monk, or nun eats meat, in direct violation of the traditions and ascetic, ecclesiastical requirements of the Priscillian sect. To do this is considered a grave sin and makes the cleric unworthy of their position.;;;;;;;;;;;;;x
 defiance_of_asceticism;Defier of Asceticism;;;;;;;;;;;;;x
 defiance_of_asceticism_desc;This character is decadent, lives in excess and/or in counter to their faith's paramount virtues of asceticism. [Root.GetHerHis] makes a mockery of their religion and has earned the ire and disgust of all virtuous members of [Root.GetHerHis] faith.;;;;;;;;;;;;;x
+#### From customizable localisation ;;;;;;;;;;;;;;x
+Actor;Actor;;;;;;;;;;;;;x
+Actress;Actress;;;;;;;;;;;;;x
```
Comment (loup99) [inline WTWSMS/localisation/WtWSMS_misc.csv]: Not from WtWSMS: this is from ACR.

Comment (IhateTrains) [inline WTWSMS/localisation/WtWSMS_misc.csv]: moved to ACR_other

**Comment by IhateTrains** [comment]
Total of Validator errors after Part I: 8459

**Pull request merged**